lib/StripeResource.js

```diff
@@ -50,7 +50,7 @@ StripeResource.prototype = {
   // Methods that don't use the API's default '/v1' path can override it with this setting.
   basePath: null,
 
-  initialize: function() {},
+  initialize() {},
```

Comment:
I'd either revert this or change it to `initialize: () => {},`

Reply:
Yeah, I prefer that style too... Doesn't seem lebab supports it, I think. Will try to see if I can find an eslint autofix...